src/main.ts

```diff
@@ -0,0 +1,39 @@
+const userInput1: HTMLInputElement = document.getElementById('userInput1') as HTMLInputElement;
+const userInput2: HTMLInputElement = document.getElementById('userInput2') as HTMLInputElement;
+const operationType: HTMLSelectElement = document.getElementById('operationType') as HTMLSelectElement;
+const answer: HTMLElement = document.getElementById('answer')!;
+const resultBtn: HTMLButtonElement = document.getElementById('resultBtn') as HTMLButtonElement;
+
+function getResult():void {
+    const firstInput: number = Number(userInput1.value);
+    const secondInput:number = Number(userInput2.value);
+    const operator:string = operationType.value;
+
+    const result:number = calculate(firstInput, secondInput, operator);
+    answer.innerText = result.toString();
+}
+
+function calculate(firstInput:number, secondInput:number, operator:string):number {
```

Comment:
요거 인터페이스로 만들면
`interface CalculatorInput {
    firstInput: number;
    secondInput: number;
    operator: string;
}
`
그래서 정리하면,
```suggestion
function calculate(firstInput, secondInput, operator): CalculateInput{
```